pkg/network/ebpf/c/tracer-stats.h

```diff
@@ -46,6 +47,47 @@ static __always_inline void update_conn_state(conn_tuple_t *t, conn_stats_ts_t *
     }
 }
 
+static __always_inline bool _isTLS(conn_tuple_t *t) {
```

Comment:
let's add documentation for this function and the other function, so it will be clear what each of them is doing

Reply:
done